services/utils/apiRequests.js

```diff
@@ -0,0 +1,32 @@
+import { clientEnv } from "@config/schemas/clientSchema";
+
+export async function sendRequest(requestConfig) {
+  const url = `${clientEnv.NEXT_PUBLIC_BASE_URL}${requestConfig.url}`;
+  try {
+    const res = await fetch(url, {
```

Comment:
Could we perhaps have a mapper function on this file for the Request options?

Something like:
GetRequestOptions(requestConfig) {
// run all the logic in here instead of adding it in the sendRequest method?
}

Reply:
Certainly, I appreciate your idea 👍